examples/lorawan/Makefile

```diff
@@ -25,8 +38,20 @@ USEPKG += semtech-loramac
 USEMODULE += $(DRIVER)
 USEMODULE += fmt
 FEATURES_REQUIRED += periph_rtc
+FEATURES_REQUIRED += periph_eeprom
```

Comment:
This should be feature required if `pm` is used. Otherwise it should still be optional. I wouldn't force its usage for the example, it limits the amount of boards this example will apply for.